packages/iframe-stamper/src/index.ts

```diff
@@ -50,9 +50,15 @@ export enum IframeEventType {
   // Event sent by the parent to establish secure communication via MessageChannel API.
   // Value: MessageChannel port
   TurnkeyInitMessageChannel = "TURNKEY_INIT_MESSAGE_CHANNEL",
-  // Event sent by the parent to get the target embedded key's public key.
-  // Value: public key
+  // Event sent by the parent to get the iframe target embedded key's public key.
+  // Value: none
   GetEmbeddedPublicKey = "GET_EMBEDDED_PUBLIC_KEY",
+  // Event sent by the parent to reset the iframe's embedded key.
+  // Value: none
+  ResetEmbeddedKey = "RESET_EMBEDDED_KEY",
```

Comment:
What do you think about naming this `ClearEmbeddedKey` instead? That'll make the comment more understandable, and SDK clients won't need to comment _their_ code to explain that `ResetEmbeddedKey` actually only clears it. We stop the bleeding here :D

Reply:
love it!